src/bundle/Controller/ContentTypeController.php

```diff
@@ -36,10 +37,13 @@
 use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
 use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentTypeDraft;
 use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentTypeGroup;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\Query\ContentTypeQuery;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\Query\Criterion\ContentTypeGroupId;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\Query\SortClause\Name;
 use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
 use Ibexa\Core\MVC\Symfony\Security\Authorization\Attribute;
+use function Ibexa\PolyfillPhp82\iterator_to_array as iteratorToArray;
```

Comment:
The point is to NOT alias the function, so that removing the polyfill is as easy as removing the `use` section.
```suggestion
use function Ibexa\PolyfillPhp82\iterator_to_array;
```